tests/pvc_utils.go

```diff
@@ -0,0 +1,74 @@
+package tests
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	defaultTimeout = 30*time.Second
+)
+
+// Creates a standard PVC in the passed in namespace.
+func CreateDefaultPVC(client *kubernetes.Clientset, namespace string, name string, size string) *k8sv1.PersistentVolumeClaim {
+	return CreatePVCFromDefinition(client, namespace, NewPVCDefinition(name, size, nil))
+}
+
+// Creates a PVC in the passed in namespace from the passed in PersistentVolumeClaim definition.
+func CreatePVCFromDefinition(client *kubernetes.Clientset, namespace string, def *k8sv1.PersistentVolumeClaim) *k8sv1.PersistentVolumeClaim {
+	var pvc *k8sv1.PersistentVolumeClaim
+	var err error
```

Comment:
`err` doesn't need to be declared here.

Reply:
Right, I only use it in the scope of the poll, so I can declare it there. I can't do pvc, err := because that would make the pvc assignment scoped in the if block and won't assign it for when I want to return.